chroma-agent/chroma_agent/lib/agent_teardown_functions.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2017 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a MIT-style
+# license that can be found in the LICENSE file.
+
+
+agent_daemon_teardown_functions = []
+
+
+def agent_daemon_teardown_function():
+    """
+    A decorator for connecting method to teardown. A method decorated with this will be called each time the
+    daemon is run. The call will occur just before the agent starts receiving commands from the manager.
```

Comment:
Does this line of the comment need updating to reflect it's teardown, not startup?